pytradfri/api/aiocoap_api.py

```diff
@@ -119,10 +120,10 @@ def _execute(api_command):
         return api_command.result
 
     @asyncio.coroutine
-    def request(*api_commands):
+    def request(api_commands):
         """Make a request."""
-        if len(api_commands) == 1:
-            result = yield from _execute(api_commands[0])
+        if not isinstance(api_commands, collections.Iterable):
```

Comment:
Please add a test for this. I would explicitly check for lists, because generators can be used for async in python 3.4